Source/WebCore/rendering/RenderTable.cpp

```diff
@@ -1541,6 +1549,22 @@ std::optional<LayoutUnit> RenderTable::firstLineBaseline() const
     return std::optional<LayoutUnit>();
 }
 
+std::optional<LayoutUnit> RenderTable::lastLineBaseline() const
+{
+    if (isWritingModeRoot() || shouldApplyLayoutContainment())
+        return std::optional<LayoutUnit>();
+
+    recalcSectionsIfNeeded();
```

Comment:
shouldn't lastBaseline be called when we are already done with laying out the section? I am just curious of why this call is needed here.

Reply:
I wasn't entirely sure about that myself so I just decided to keep the logic similar to what was already in firstLineBaseline. I see that `setNeedsSectionRecalc` gets called in a couple different places like `removeColumn`, which gets called from `willBeRemovedFromTree`. I'm not familiar with the scenarios in which can occur. I can remove it if you would like or we can just keep it to remain consistent for now.